### ISSUE TITLE
LSOA and MSOA codes

### ISSUE THREAD

**Issue by peterdutey** [ideal-postcodes/postcodes.io#602]
Hello

This is user feedback/suggestion. 
I'm a huge fan of postcode.io but there is one thing that puzzles me is that you seem to pull the LSOA and MSOA names rather than codes from the ONSPD. What's more, the codes are not included under /codes/
This makes linkage with a lot of other data more difficult.

Another two related aspects are that:
- having the output areas too would be of use to some users
- the census next year means ONSPD is soon going to provide lsoa21 rather than lsoa11, msoa21, etc. What I mean is there are likely to be breaking changes in any case which might not happen at the same time in England, Scotland, Wales and NI.

I couldn't track the relevant chunk in your source code at the first glance but I'd be happy to spend a bit more time on it if you agree with this comment in principle.
Just let me know!

Best,
Peter

### Example below

[http://api.postcodes.io/postcodes/SW1A2AA ](http://api.postcodes.io/postcodes/SW1A2AA)
```
{"status":200,"
result":{
"postcode":"SW1A 2AA",
// OMITTING DETAIL
"lsoa":"Westminster 018C",
"msoa":"Westminster 018",
// OMITTING DETAIL
"codes":{"admin_district":"E09000033",
"admin_county":"E99999999",
"admin_ward":"E05000644",
"parish":"E43000236",
"parliamentary_constituency":"E14000639",
"ccg":"E38000031",
"ccg_id":"09A",
"ced":"E99999999",
"nuts":"UKI32",

// ***Expected: ***
"oa11":"E00023936",
"lsoa11": "E01004736",
"msoa11": "E02000977"

}
}}
```

**Comment by cblanc** [comment]
Sure, will schedule for ONSPD Nov 2020

**Comment by cblanc** [comment]
This is available in [13.0.0](https://github.com/ideal-postcodes/postcodes.io/releases/tag/13.0.0)

**Comment by cblanc** [comment]
Now also available on the public API
